game.lua

```diff
@@ -472,6 +472,14 @@ function formatNumber(n)
   return n
 end
 
+function float(n, digits)
```

Comment:
I'm not sure what does this function do - and couldn't find a observable change in the game also. Can you tell the purpose of this, please?

Reply:
Aha. It was related to this issue I had closed:
https://github.com/Seniru/TFM_Clicker/issues/25

Reply:
Oh OK

Reply:
You may also call this function inside formatNumber to make the thing more easy

Reply:
And I reopened #25 and will close after  merging this.
(Add a commit titled closes #25 to automatically close it)